### PR TITLE
fix message format

### PR DESCRIPTION
**Pull request by monsteredc**
**What type of PR is this?**
/kind bug

**What this PR does / why we need it**:
The message string seems not to be formatted, just format the parameters.

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

File: pkg/stream/message.go [hunk 1]
```diff
@@ -76,7 +76,7 @@ func (m *Message) Bytes() []byte {
 }
 
 func (m *Message) String() string {
-	return fmt.Sprintf("MESSAGE: connectid %v MessageType %s", m.ConnectID, m.MessageType)
+	return fmt.Sprintf("MESSAGE: ConnectID %v MessageType %s", m.ConnectID, m.MessageType)
```
Comment (subpathdev) [inline pkg/stream/message.go:79]: can you use either `connectID` or `ConnectID` i would preffer `connectID`

Comment (monsteredc) [reply pkg/stream/message.go]: @subpathdev Done. Thanks for the suggestion!

**Review by subpathdev — APPROVED** [review]

**Review by fisherxu — COMMENTED** [review]
/approve

**Pull request merged**